index.html

```diff
@@ -13,139 +13,154 @@
 </head>
 
 <body>
-    <header class="container-header">
-        <a id="name" href=""><img class="mylogo" src="attributes/mylogo.png" alt="logo" /></a>
-        <nav><img class="union" src="attributes/union.png" alt="union_icon" /></nav>
-        <nav class="desktop_headicons">
-            <a class="header_btn" href="">Portfolio</a>
-            <a class="header_btn" href="#about_text">About</a>
-            <a class="header_btn" href="#full_name">Contact</a>
-            <a class="header_btn" href=""><img id="mail" src="attributes/mailicon.png" alt="mailicon" /></a>
+    <header>
+        <nav class="desktop-nav">
+            <a id="name" href=""><img class="mylogo" src="attributes/mylogo.png" alt="logo" /></a>
```

Comment:
- [ ] Kindly hide the logo from inside the mobile menu to match the design in Figma. Also, kindly add `border-bottom` to all links to match the design 👍 

## Your design:
![image](https://user-images.githubusercontent.com/50721479/201729575-8a45e46d-4204-41ce-bdb1-93a5bed1c72d.png)

## Figma design:
![image](https://user-images.githubusercontent.com/50721479/201729609-af1c3a5e-c814-4cbf-b948-ae3831d5811f.png)